lib/elixir_sense/providers/suggestion.ex

```diff
@@ -4,13 +4,12 @@ defmodule ElixirSense.Providers.Suggestion do
   """
 
   alias Alchemist.Helpers.Complete
-  alias ElixirSense.Core.Introspection
-
-  alias ElixirSense.Core.TypeInfo,
-        alias(ElixirSense.Core.State.{StructInfo})
 
+  alias ElixirSense.Core.Introspection
```

Comment:
Can you actually alphabetize these while you're here?

Reply:
sure thing, now should be sorted 👍

these things are hard to maintain without a credo to make sure of it… will try seeing about that in another branch soon.

cheers! 🍷